natural_resources/src/main.py

```diff
@@ -0,0 +1,29 @@
+"""Load all intermediate datasets and generate natural resources dataset.
+
+"""
+
+from natural_resources.src import eia
+from natural_resources.src import OUTPUT_DIR
+
+
+def extract():
+    eia.generate_yearly_dataset()
+    eia.generate_monthly_dataset()
+
+
+def transform():
+    pass
+
+
+def load():
+    pass
+
+
+def main():
```

Comment:
Was there a rationale behind leaving `transform` and `load` here?

Reply:
Hi @edomt, I simply copied the format from the climate_change folder. I thought we would add more datasets on natural resources, that would be merged here before producing a final dataset (to be stored in the 'output' folder). If for the moment that's not the plan, I can delete these redundant lines.

Reply:
But my sense was that @HannahRitchie wanted to already build a first small explorer just based on the EIA data – in which case there needs to be already some files in `output` to use in that explorer? (But maybe I misunderstood and she wants to bring in more data from other sources before creating a final file)

Reply:
I think you are right, however, the files she's using on the explorer come from the 'ready' folder. I can either:
1. Move the data from 'ready' to 'output' (if we believe we are not going to add anything new for the time being).
2. Copy the data from 'ready' to 'output' (if we believe that we will end up adding more resources apart from fossil fuels).

Reply:
Yes, later we will add more metrics to this. But our plan is to first publish (hopefully tomorrow) with EIA fossil fuel data.

For this, all I need is a link to a csv (which I have). The location of this is not relevant to me, so wherever makes sense on the side of the data team.

Reply:
Let's keep it this way for now - for now it doesn't make much sense to write a merging step based only on one source anyway 🤷🏻‍♂️